src/vehicle.cpp

```diff
@@ -3249,7 +3286,7 @@ float vehicle::k_friction() const
 {
     // calculate safe speed reduction due to wheel friction
     float fr0 = 1000.0;
-    float kf = ( fr0 / (fr0 + wheels_area()) );
```

Comment:
Ideally return these to they way they were originally so they don't appear in the diff at all.